docs/apis/cluster_execution.md

```diff
@@ -98,7 +98,7 @@ The latter version is recommended as it respects the classloader management in F
 
 To provide these dependencies not included by Flink we suggest two options with Maven.
 
-1. The maven assembly plugin builds a so called fat jar cointaining all your dependencies.
+1. The maven assembly plugin builds a so-called Fat jar(executable jar) containing all your dependencies.
```

Comment:
Prefer to use term uber-jar as mentioned in the maven assembly page [1]

[1] http://maven.apache.org/plugins/maven-assembly-plugin/